packages/carequality-sdk/src/models/bundle.ts

```diff
@@ -0,0 +1,21 @@
+import { z } from "zod";
+import { identifierSchema, meta, objectNumericValue, objectValue } from "./shared";
+import { organizationSchema } from "./organization";
+
+export const bundleSchema = z.object({
```

Comment:
Why don't we call this `stu3BundleSchema` to be explicit, and not confuse this with the R4 `Bundle` we use elsewhere?